client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -96,17 +96,19 @@ public final class ApiEndpointsV1
     public static final String KEYSPACE_TOKEN_MAPPING_ROUTE = API_V1 + PER_KEYSPACE + "/token-range-replicas";
 
     // Blob Transport Extension
-    public static final String JOB_ID_PATH_PARAM = ":jobId";
+    // todo: refactor other path param use cases; and replaceAll in requestURI building
+    public static final PathParam JOB_ID_PATH_PARAM = PathParam.of("jobId");
```

Comment:
I think this is overly complex, maybe it deserves to be its own patch if you wanted to change it. I'd keep it simple here following the existing pattern